spark/v3.3/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/parser/extensions/IcebergSqlExtensionsAstBuilder.scala

```diff
@@ -92,14 +94,17 @@ class IcebergSqlExtensionsAstBuilder(delegate: ParserInterface) extends IcebergS
       typedVisit[Transform](ctx.transform))
   }
 
+  /**
+   * Create an CREATE OR REPLACE BRANCH logical command.
+   */
```

Comment:
nice catches! But can we avoid unrelated changes in the PR? @amogh-jahagirdar I think you should fix these separately

Reply:
btw I am surprised that checkstyle did not catch that extra space, maybe it needs some more configuration.

Reply:
For scala, our checkstyle needs to be improved. And auto-formatting doesn't work yet.

Reply:
Great catch @hililiwei , I'll update #6728 to fix the styling issues. We should track an issue for scala checkstyle/autoformatting , admittedly I rely on that quite heavily to catch these kinds of issues.

Reply:
Created https://github.com/apache/iceberg/issues/6736 and updated #6728 to include styling fixes

Reply:
Thanks, I merged #6728